### ISSUE TITLE
Show remaining conflicts with CiviCRM 5.19+

### ISSUE THREAD

**Issue by bjendres** [systopia/de.systopia.xdedupe#1]
A change in the API by @eileenmcnaughton returns any remaining merge conflicts with CiviCRM ``5.19`` and later (I think). The system should expose this to the user

**Comment by eileenmcnaughton** [comment]
5.18 I think

**Comment by bjendres** [comment]
I found this:
``` 
% git tag --contains 38f02a81d4b5e239f4f434d9bf42a7a2fa0a3f8f
5.16.0
5.16.1
5.16.2
5.16.3
5.16.4
5.17.0
5.17.1
5.17.2
5.17.3
5.17.4
5.17.5
5.18.0
5.18.1
```

That would be ``5.16``. @eileenmcnaughton: feasible?

**Comment by eileenmcnaughton** [comment]
@bjendres it started in 5.16 but matured by 5.18 I would say

**Comment by bjendres** [comment]
Thanks, I adjusted it.

**Comment by bjendres** [comment]
released with ``0.1``
